CHANGES.md

```diff
@@ -7,6 +7,7 @@ New styles:
   none.
 
 Improvements:
+- enh(parser): adds `data-language` attribute to highlighted blocks (#1708) [canvasplay][]
```

Comment:
I'm worried that "data-language" is overly generic. maybe we namespace this as "data-hljs-language"? The will reduce the risk of conflict, and allows us to more easily add more attributes in the future while minimizing the risk of collisions with existing attributes.

Reply:
I say “wait and see” and we can add a prefix option later if necessary.

Reply:
This might catch people off tho... imagine some random application that depends on our library already does the following:

```
document.querySelectorAll(['data-language']).forEach(doStuff)
```

(You can see just on [github search](https://github.com/search?q=document.querySelectorAll%28%5B%27data-language%27%5D%29&type=Code) examples)

By adding just 'data-language', we've just unexpectedly got added to someone's unrelated code potentially breaking someone's application. The risk of that is reduced by the "hljs-"  prefix. 

Again, there are [zero hits](https://github.com/search?q=document.querySelectorAll%28%5B%27hljs-data-language%27%5D%29&type=Code) on Github search for `data-hljs-language`. 

And I only found 2 hit on Google: 

https://github.com/pc035860/angular-highlightjs/issues/33

And: 
![Screenshot 2019-11-14 14 32 04](https://user-images.githubusercontent.com/870154/68824614-8e239800-06eb-11ea-9e9f-2623c95037f6.png)

Reply:
Well, now I thought people would actually look at the pieces they were putting together and make sure they were all compatible.  I'm not sure the best thing is for every program to preemptively assume a namespace is needed... that sounds very ugly.  Though I understand the reasoning behind it.

But you've persuaded me that perhaps the whole idea here was accepted a bit too quickly... Things already in-flight (PRs) before I started doing triage got a lot lighter review than new requests... I think this is perhaps better handled with a plugin, and makes a good use case for a post highlightBlock plugin.

https://github.com/highlightjs/highlight.js/pull/2285

Reply:
> Well, now I thought people would actually look at the pieces they were putting together and make sure they were all compatible.

As you’ve eloquently stated previously: lol. 

> I think this is perhaps better handled with a plugin, and makes a good use case for a post highlightBlock plugin.

I think this still has merit/utility when automatic language detection occurs. 

Re namespace: Note also we use the namespace in our CSS. This follows the same principle to minimize the chance of collisions.